frontend/src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import '../MyNavbar/MyNavbar.css'
+import {Button} from 'reactstrap';
+
+
+
+//Compact button component
+//Set prop 'target' to "_blank" to open the link in a new tab 
+function LinkButton(props) {
+    return(
+        <>
+        <Button color="info" className="button-background" href = {props.url} target={props.target}>
```

Comment:
I think the color="info" is overriding the background color for the button. I was thinking that maybe the color="info" could be replaced with style={{ background: 'rgb(176, 216, 201)' }} instead to directly edit the background color of the button in the Button tag!

Reply:
Thank you! I could not figure out a way to do this so I just left the info color. I changed it and pushed the changes.
![image](https://user-images.githubusercontent.com/62858601/99744546-84223b00-2a9d-11eb-8d5d-6cd62a5b8476.png)

Reply:
That's perfect! I like how you made the text inside the button match the navbar color around it!